core/src/main/java/hudson/util/XStream2.java

```diff
@@ -415,7 +415,7 @@ public Object unmarshal(HierarchicalStreamReader reader, UnmarshallingContext co
 
         private PluginManager pm;
 
-        @SuppressWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE") // classOwnership checked for null so why does FB complain?
+        @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE") // classOwnership checked for null so why does FB complain?
```

Comment:
I agree, though it should have been done when the annotations was introduced initially. I do not think the inline comment is a good `justification` btw.